tests/travis/docker-run-mysql-or-mariadb.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -ex
+
+echo "Starting RDBMS…">&2
+
+if [[ "$IMAGE" == "mysql:8.0" ]]
+then
+  CMD_OPTIONS="--default-authentication-plugin=mysql_native_password"
+else
+  CMD_OPTIONS=""
+fi
+
+docker run \
+    --health-cmd='mysqladmin ping --silent' \
+    --detach \
+    --env MYSQL_ALLOW_EMPTY_PASSWORD=yes \
+    --env MYSQL_DATABASE=doctrine_tests \
+    --publish 33306:3306 \
+    --name rdbms \
+    "$IMAGE" $CMD_OPTIONS
+
+while true; do
+  healthStatus=$(docker inspect --format "{{json .State.Health.Status }}" rdbms)
+  case $healthStatus in
+    '"starting"')
+      echo "Waiting for RDBMS to become ready (status = $healthStatus)…">&2
```

Comment:
Since you're using the default case to handle unknown statuses, displaying the status here is unnecessary (it’s always the same).